src/dlgRoomExits.cpp

```diff
@@ -1481,6 +1521,15 @@ void dlgRoomExits::init()
     }
 
     roomID->setText(QString::number(mRoomID));
+    if (pR->isLocked) {
+        // Add the padlock icon to the room number display to show the locked
+        // status of the room:
+        roomID->addAction(mpAction_exitRoomLocked, QLineEdit::TrailingPosition);
+        // Also revise the tool tip:
+        roomID->setToolTip(utils::richText(tr("This is the Room ID number for this room; this <b>room is locked</b> so it will not be used for speed-walks at all. Neither of these details can be changed here!",
```

Comment:
```suggestion
        roomID->setToolTip(utils::richText(tr("This is the Room ID number for this room; this <b>room is locked</b> so it will not be used for speed-walks at all.",
```

Reply:
👎 It isn't clear from the `QLineEdit` widget used that it is set to read-only mode - this text (and it's predecessor in the prior code), and the "Forbidden" cursor, is used to give the end-user a clue why they cannot change the room's number.

Reply:
I don't think this is still helpful. Let's wait for feedback from the field - eg people being confused about this - before adding it. The fact that it's read only is enough of a hint, a forbidden cursor is going overboard and a tooltip on top of that is just not aesthetic anymore imho.

Reply:
A `QLineEdit` that is set to **read-only** mode is NOT visibly different from a normal one that can be written in. (This is NOT the same as the appearance when it is *disabled*!) This is why we have some horrible, incompatible with style-sheets, messing about with `QPalette`s to change the colouration for some of the `QLineEdit`s in the "Connect profile" dialogue BTW. Therefore, providing hints, in the form of the "forbidden" cursor and explaining that the entry cannot be changed - ***has been present on that field for many years already*** and is NOT being added here!

Reply:
What do others think? @demonnic @keneanung

Reply:
Is there something other than a `QLineEdit` widget that can be used to display this in a way that is obviously not editable? That seems to me like it'd be the best solution. Then the tooltip would definitely not be needed. I admit I'm not super familiar with the widgets available though.

Reply:
:thinking: I suppose we could just use `QLabel`s for those - either just by extending the title labels (the red things below) to the left of the `QLineEdit`s) to display the value as well as the title - or adding additional ones to display the values only - the latter will probably be simpler to do to maintain the alignments (right for the titles, left for the values):

![Screenshot_20220616_233110](https://user-images.githubusercontent.com/6163092/174189588-650559f6-f49a-4d2e-898a-318f2f9b9b24.png)

Reply:
Seems like something that could work - could you push the change so we can preview it live?

Reply:
Done in 3f8b8c1 .

Reply:
Looks fine, but the tooltip is still a bit intense, no?

![image](https://user-images.githubusercontent.com/110988/174425100-70875940-7fc7-4ba4-bcda-ba5ca9a7fcbf.png)

Reply:
I think not, as it happens it is ***not*** possible to change the Room ID *anywhere* but given how many other things change be modified through this dialogue I do not believe it to be *that* excessive...

BTW Have you inspected how things look when the room is locked?

Reply:
Yes, - 
![image](https://user-images.githubusercontent.com/110988/174474725-081b61ec-3dcb-40ed-860d-f162abc12507.png)

Reply:
I still think we should apply the original suggestion - it's more concise.